uploader/templates/upload.html

```diff
@@ -318,6 +322,47 @@ <h5 class="card-title text-center">
             },
             width: 80
           },
+                    {
+            title: "Categories",
+            field: "categories",
+            formatter: function(cell, formatterParams, onRendered) {
+              return `<select name="wpCategories" id="wpCategories${
+                cell.getData()["id"]
+              }" class="custom-select" value="">
+  
+<option title="Abstract" value="Abstract">Abstract</option>

```

Comment:
I dont know if we really want to hardcode Categories though ;-(

Reply:
I'm sorry @tonythomas01 
I don't know how but the changes that i made for categories have been added into this branch and I would request you to ignore this branch and review PR#59
Thanks!